resources/com.yubico.yubioath.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.yubico.yubioath</id>
+  <launchable type="desktop-id">com.yubico.yubioath.desktop</launchable>
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+
+  <name>Yubico Authenticator</name>
+  <summary>Graphical interface for displaying OATH codes with a YubiKey</summary>
+  <url type="homepage">https://developers.yubico.com/yubioath-desktop/</url>
```

Comment:
It could use a bugtracker url

Reply:
It is not a field for issue tracker, it is homepage

Reply:
This repository, as well as the URL to open a support ticket ( https://support.yubico.com/support/tickets/new  ) is fine to include.

Reply:
Replaced ccad977